docs/download/downloading_data.md

```diff
@@ -0,0 +1,261 @@
+# Downloading Data
+
+## Downloading data from Collaboratory repository
+
+Academic research cloud infrastructure built to house ICGC data.
+
+Repository information can be found [here](https://docs.icgc.org/download/repositories/#collaboratory)
+
+#### Prerequisites
+
+1.  [Apply for DACO access](https://github.com/icgc-dcc/dcc-docs/blob/download-doc-update/docs/download/data-access.md#apply-for-access-to-controlled-data)
+2.  Download and install score-client software [Tarball](/software/download/#score-client), [Docker](https://hub.docker.com/r/overture/score/) (Download/configuration instructions [here](https://docs.icgc.org/download/guide/#installation-of-the-score-client))
+
+<span style="color:blue">NOTE: We are currently actively copying controlled data from [EGA](https://ega-archive.org) to Collaboratory so it can be available to download from either repository. This is an on-going process, so we do not currently have all the EGA datasets in Collaboratory yet.</span>
+
+#### Download Data Instructions
+
+1.  Once your DACO application has been approved, use your OpenID (ie. the Gmail email address you specified in your DACO application) to log into [https://dcc.icgc.org/](https://dcc.icgc.org)
+
+    ![Portal-Login](images/Portal_login.png)
+
+2.  After successful authentication, you will know that you have Cloud Access to the controlled tier if the `Login` link is replaced with a green cloud icon
```

Comment:
Should we mention about both icons (DACO + Cloud). Don't know whether it ever comes up in help tickets, if they if a user may have DACO , but not CLOUD?

Reply:
Good point Aleks. I updated the docs to reflect this